pkg/duplicateevents/duplicated_event_patterns.go

```diff
@@ -40,6 +40,10 @@ const (
 	ProbeErrorConnectionRefusedRegExpStr      = "reason/ProbeError.*Readiness probe error.*connection refused"
 	SingleNodeErrorConnectionRefusedRegExpStr = "reason/.*dial tcp.*connection refused"
 
+	ErrorReconcilingNode  = "reason/ErrorReconcilingNode roles/worker *annotation not found for node"
+	FailedScheduling      = "reason/FailedScheduling *nodes are available"
+	OperatorStatusChanged = "reason/OperatorStatusChanged Status for *changed"
+
```

Comment:
Sorry I missed this in the review yesterday but I think you are still missing the part where the new test is created.  Looks like these are added to AllowedRepeatedEventFns but there should be an accompanying test.  

You can do a Find Usages on 'ProbeErrorConnectionRefusedRegExpStr' and see how it is also used in a test case https://github.com/deepsm007/origin/blob/89fd9a415c71772d0377ecd6eeedc89752ee3288/pkg/synthetictests/apiserver.go#L46

Reply:
Does adding them here result in a new test somehow? These look pretty serious to ignore as pathological events. Where did these three come from? Were they high offenders and if so, how common?